web/src/main/java/com/epam/rft/atsy/web/configuration/WebConfiguration.java

```diff
@@ -97,4 +98,9 @@ protected void configureHandlerExceptionResolvers(
     // The index is 1, in order to add it after the ExceptionHandlerExceptionResolver
     exceptionResolvers.add(1, new UncheckedExceptionResolver(mappingJackson2JsonView()));
   }
+
+  @Bean(name = "multipartResolver")
```

Comment:
name annotation attribute is unnecessary, because the bean's name is the method name by default, in this situation the method name and the value of the name attribute are the same